packages/forklift-console-plugin/src/components/mappings/MappingPage.tsx

```diff
@@ -58,43 +58,43 @@ const byName = {
   isVisible: true,
   filter: {
     type: 'freetext',
-    toPlaceholderLabel: (t) => t('Filter by name'),
+    toPlaceholderLabel: 'Filter by name',
```

Comment:
Do we really drop translating labels in our plugin?

Reply:
see comment above about removing translation from common package

Reply:
Not sure if this applies here. This structure is in the plugin package.

Reply:
ahh, missed the "in our plugin?" :-)

fixing :+1: